<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Department</class>
 <widget class="QMainWindow" name="Department">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Department</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 186, 186);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>361</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="btn_edit">
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>EDIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Delete">
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>DELETE</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Rockwell Condensed&quot;;
color: rgb(0, 0, 127);</string>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NAME</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">image: url(:/newPrefix/Aydin.png);
image: url(:/newPrefix/Aydin.png);
image: url(:/newPrefix/Aydin.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../Documents/masaüstü/asd.qrc"/>
 </resources>
 <connections/>
</ui>
